controller/bindings/connman/connman.mli

```diff
@@ -38,21 +38,20 @@ end
     A D-Bus ConnMan agent is implemented by this module to provide inputs for secured networks. The agent is not started manually, but is automatically created by the [Service.connect] function.
 *)
 module Agent : sig
-
   (** Input that the agent may provide to connect to a network.
 
       Note that not all possible inputs are supported and thus connecting to some networks is not possible (e.g. WPS). See the ConnMan Agent API documentation for more information.
   *)
   type input =
-    | None (** No input *)
-    | Passphrase of string (** The passphrase for authentication. For example a WEP key, a PSK passphrase or a passphrase for EAP authentication methods.*)
+    | None  (** No input *)
+    | Passphrase of string
+        (** The passphrase for authentication. For example a WEP key, a PSK passphrase or a passphrase for EAP authentication methods.*)
```

Comment:
Maybe putting comments above would yield less weird formatting?

Reply:
Putting it above actually yields a warning: `Warning 50 [unexpected-docstring]: unattached documentation comment `